src/NHSD.BuyingCatalogue.Ordering.Services/NHSD.BuyingCatalogue.Ordering.Services.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+  </PropertyGroup>
```

Comment:
Can you add this element to the property group?

```xml
<AnalysisMode>AllEnabledByDefault</AnalysisMode>
```

Reply:
Added `AnalysisMode` element